www/css/app.css

```diff
@@ -89,6 +90,32 @@
     margin: 0 1rem;
 }
 
+/* App theme: dark */
+
+.dark header, .dark #cachingAssets, .dark #about, .dark #configuration, .dark #welcomeText, .dark #articleListWithHeader,
+.dark #alertBoxHeader, .dark footer, .dark img {
+    filter: invert(1) hue-rotate(180deg);
+}
+
+.dark body {
+    background: #171717;
+}
+
+/* End of app theme: dark */
+
+/* Content themes: _invert, _mwInvert */
+
+iframe._invert, iframe._mwInvert {
+    filter: invert(1) hue-rotate(180deg);
+}
+
+/* End of content themes */
+
+.hidden {
+    display: none;
+    margin: 5px;
```

Comment:
Why is this margin necessary? The class name is very generic and does not indicate that it adds a margin

Reply:
@mossroy The margin helps to position these elements in relation to each other. Originally I intended this to be a generic class for any hidden element, but it evolved into positioning as well. I'll rename to something related specifically to these links.